tests/sentry/issues/auto_source_code_config/test_frame_info.py

```diff
@@ -12,7 +12,7 @@
 
 UNSUPPORTED_FRAME_FILENAMES = [
     "async https://s1.sentry-cdn.com/_static/dist/sentry/entrypoints/app.js",
-    "/gtm.js",  # Top source; starts with backslash
+    "/gtm.js",
```

Comment:
This fails because it will be treated as a single file.

I need to add a test case for when we have at least a directory.